tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/terraform/custom-metrics-stackdriver-adapter/main.tf

```diff
@@ -0,0 +1,291 @@
+resource "kubernetes_namespace_v1" "custom-metrics" {
```

Comment:
This file also lives in https://github.com/GoogleCloudPlatform/ai-on-gke/tree/main/benchmarks/inference-server/text-generation-inference/custom-metrics-stackdriver-adapter. Maybe this new file and the old one should just be refactored out to a module used by both?

Reply:
Actually theres quite a few duplicates in the repo, like[this one](benchmarks/inference-server/jetstream/jetstream.yaml) for example. Not very concerned with this since this is blocking since I need to reference this guide elsewhere. Following this up with a focused component refactoring PR.

Reply:
Refactoring followup sgtm.